exalearn-ui/src/app/components/test-nav-btn/test-nav-btn.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { Question } from '../../interfaces/interfaces';
+
+@Component({
+	selector: 'app-test-nav-btn',
+	templateUrl: './test-nav-btn.component.html',
+	styleUrls: ['./test-nav-btn.component.scss']
+})
+export class TestNavBtnComponent implements OnInit {
+	// @ts-ignore
```

Comment:
Same here, try get rid of ts-ignores

Reply:
I understand that using ts ignores is bad so i think we should change ts config to work with inputs